packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks Extension v1.2
+
+## What's new?
+
+-   [📗 Advanced notebook support with Databricks Connect V2](#dbconnect-notebook)
```

Comment:
We really should not use V2 in our docs. It was an internal thing so that we could distinguish it from V1 at first. But now that it has escaped into the wild, a few customers have been complaining about where to find "v2.0.0" of Databricks Connect, which doesn't actually exist. Instead, we've been using something like "Databricks Connect for Databricks Runtime 13.0 and above" on first mention, and then once that context is clear, shortening simply to "Databricks Connect" on subsequent mentions. You can also keep the short version of "Databricks Connect" in titles, with the first mention in the body being "Databricks Connect for Databricks Runtime 13.0 and above" for clarity.